poetry/console/commands/env/list.py

```diff
@@ -15,8 +15,14 @@ def handle(self):
 
         manager = EnvManager(self.poetry)
         current_env = manager.get()
-
-        for venv in manager.list():
+        env_list = manager.list()
+        if (
+            self.poetry.config.get("virtualenvs.in-project")
+            and current_env not in env_list
+        ):
+            env_list.insert(0, current_env)
+
```

Comment:
Thanks a lot for your contribution. :+1: 

I wonder if this check shouldn't be implemented directly in the `list()` method.

Reply:
Makes sense, PR updated.